pxtlib/package.ts

```diff
@@ -380,8 +380,10 @@ namespace pxt {
             // no core package? add the first one
             if (corePackages.length == 0) {
                 const allCorePkgs = pxt.Package.corePackages();
+                /* tslint:disable:no-unused-expression TODO(tslint): */
                 if (allCorePkgs.length)
                     this.config.dependencies[allCorePkgs[0].name];
```

Comment:
nice another bug...

Reply:
wait what's the bug here?

Reply:
this line does nothing but the comment seems to say otherwise.